apps/website/src/pages/docs/button.mdx

```diff
@@ -17,6 +26,46 @@ import LiveExample from '~/components/LiveExample.astro';
 
 Buttons are the recipients of a user onClick action (a mouse click for browsers and a finger press on touch-sensitive devices). Buttons have states and events and should be used when a form is submitted (post) or where the button represents a clear call to action on a page. In some cases, buttons can also be used in place of href links to draw special attention to that action.
 
-## Props
+## Button
 
-<PropsTable path={frontmatter.propsPath} />
+<LiveExample src={frontmatter.exampleCodeFileDefault} />
+
+### Props
+
+<PropsTable path={frontmatter.propsPathDefault} />
+
+## Split button
+
+<LiveExample src={frontmatter.exampleCodeFileSplitButton} />
+
+Pressing this left side of the button will commit the action. Pressing the right side of the button will open a menu. Upon selecting an option from the menu, the action immediately takes place, and the left side of the button’s label will change to match the recent selection.
```

Comment:
> left side of the button’s label will change to match the recent selection

To confirm, is this valid? Because I don't believe the left button label changes when we click on a menu option.

Reply:
i believe the wording was copied over from the old page (splitbutton.mdx). it doesn't really matter for this PR

eventually we will validate the wording of every page